### ISSUE TITLE
Doesn't properly generate input placeholder rules

### ISSUE THREAD

**Issue by necolas** [postcss/autoprefixer#189]
source:

``` css
input::placeholder { color: black; }
input::-webkit-input-placeholder { line-height: 16px; /* fix chrome bug */ }
```

output:

``` css
input::-moz-placeholder { color: black; }
input:-ms-input-placeholder { color: black; }
input::placeholder { color: black; }
input::-webkit-input-placeholder { line-height: 16px; /* fix chrome bug */ }
```

expected: 

``` css
input::-moz-placeholder { color: black; }
input:-ms-input-placeholder { color: black; }
input::-webkit-input-placeholder { color: black; }
input::placeholder { color: black; }
input::-webkit-input-placeholder { line-height: 16px; /* fix chrome bug */ }
```

I'd expect to be able to "extend" the generated webkit rule (in the example), rather than have it completely omitted by AP.

**Comment by erichonkanen** [comment]
tldr: what do I use for placeholder? `::placeholder`?

**Comment by ai** [comment]
@erichonkanen `::placeholder`

**Comment by erichonkanen** [comment]
@ai thanks

**Comment by nikuideveloper** [comment]
reduce the padding in textbox. i am able to get the placeholder in firefox
